build/configs/rtl8730e/loadable_ext_psram/defconfig

```diff
@@ -1650,6 +1669,9 @@ CONFIG_EXAMPLE_LCD_FPS_TEST=5000
 #
 # CONFIG_EXAMPLES_TAHI is not set
 # CONFIG_EXAMPLES_WIFIMANAGER_TEST is not set
+# CONFIG_WIFIMANAGER_TEST_TRIAL=5
```

Comment:
'# CONFIG_WIFIMANAGER_TEST_TRIAL is not set
'# CONFIG_EXAMPLES_WIFIMANAGER_AP_LIST_ITEMS_COUNT is not set